ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetPrivateTransaction.java

```diff
@@ -66,11 +67,10 @@ public JsonRpcResponse doResponse(final JsonRpcRequestContext requestContext) {
       return new JsonRpcSuccessResponse(requestContext.getRequest().getId(), null);
     }
     try {
+      LOG.trace("Fetching transaction information from Enclave");
```

Comment:
nit: do we still want to reference the enclave? Or can this simply say "fetching transaction info"?

Reply:
Don't think it hurts as ultimately it is still being retrieved from the enclave.